cargo-pgx/src/command/install.rs

```diff
@@ -259,6 +269,54 @@ pub(crate) fn build_extension(
         command.arg("--all-features");
     }
 
+    fn apply_sysroot(command: &mut Command, sysroot: &Option<PathBuf>) -> eyre::Result<()> {
+        if let Some(ref sysroot) = sysroot {
+            let sysroot_str = sysroot.to_str().ok_or(eyre!("sysroot is not valid utf-8"))?;
+            let sysroot_arg = format!("--sysroot={sysroot_str}");
+            command.env("BINDGEN_EXTRA_CLANG_ARGS", &sysroot_arg);
+            command.env("PG_CFLAGS", &sysroot_arg);
+        }
+
+        Ok(())
+    }
+
+    // set PG_CONFIG
+    match cross_options {
+        CrossBuild::None | CrossBuild::Target { .. } => {
+            if let Some(pg_config) = std::env::var_os("PGX_PG_CONFIG_PATH") {
+                command.env("PG_CONFIG", pg_config);
+            }
+        }
+        CrossBuild::Host { pg_config, .. } => {
+            if let Some(ref host_pg_config) = pg_config {
+                command.env("PGX_PG_CONFIG_PATH", host_pg_config);
+                command.env("PF_CONFIG", host_pg_config);
+            }
+        }
+    }
+
+    // sysroot and target handling
+    match cross_options {
+        CrossBuild::None => {}
+        CrossBuild::Target { target, sysroot } => {
+            command.arg("--target");
+            command.arg(target);
+
+            apply_sysroot(&mut command, &sysroot)?;
+        }
+        CrossBuild::Host { sysroot, .. } => {
+            let var_names = vec!["CC", "LD", "CFLAGS", "LDFLAGS"];
```

Comment:
I think this is missing AR (although perhaps it's not needed for some reason?)
```suggestion
            let var_names = vec!["CC", "LD", "CFLAGS", "LDFLAGS", "AR"];
```

Reply:
My working code branched off of 0.5.4 doesn't have it, but generally, yes, it should be passed. I believe that AR is architecture-independent, but there could be some weird edge cases around using different toolchains between host and target that would make this not work.